src/chain.h

```diff
@@ -320,6 +324,7 @@ class CDiskBlockIndex : public CBlockIndex
             READWRITE(VARINT(nDataPos));
         if (nStatus & BLOCK_HAVE_UNDO)
             READWRITE(VARINT(nUndoPos));
+        READWRITE(hashAnchor);
```

Comment:
I suggest doing something like:
```
uint8_t tag = 0;
READWRITE(tag);
if (ser_action.ForRead() && tag != 0) {
    assert(!"For the first time running zcashd after upgrading to v1.0.0-rc1, please use the -rescan option.");
}
READWRITE(hashAnchor);
```
(The tag will never be zero if it is reading the block's ``nVersion``, because the only valid block version is 4, which has a non-zero first byte in little-endian representation.)